app/controllers/repos_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ReposController < ApplicationController
+  def index
+    github_service = GithubService.new(current_user.token)
+    @repos = github_service.find_repos
```

Comment:
@jk1dd You're accessing hashes in the view instead of objects. We want to access objects in the view instead. So yes, you'll need to create a Repo PORO.